src/Marten.Testing/MartenServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,142 @@
+using Lamar;
+using Marten.Testing.Harness;
+using Microsoft.Extensions.DependencyInjection;
+using Shouldly;
+using Xunit;
+
+namespace Marten.Testing
+{
+
```

Comment:
**SUGGESTION:** It would be nice to remove those empty lines.

Reply:
Let's please not invest time in this kind of code formatting in the test code